### PR TITLE
Polish on new launch docs

### PR DESCRIPTION
**Pull request by tristan-spaulding2**
## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/job-inputs.md [hunk 1]
```diff
@@ -93,9 +90,9 @@ to load the run config input values anywhere in the job code.
 
 ## Reconfigure a file
 
-The launch sdk can manage input values stored in config files in the job code.
+The Launch SDK can manage also input values stored in YAML or JSON config files in the job code--a common approach in many deep learning and LLM use cases.
```
Comment (tristan-spaulding2) [inline docs/guides/launch/job-inputs.md]: @KyleGoyette can we even cite Transformers etc. as an example here?

File: docs/guides/launch/job-inputs.md [hunk 2]
```diff
@@ -93,9 +90,9 @@ to load the run config input values anywhere in the job code.
...
 
-The `launch` sdk provides a way to manage input values stored in config files in the job code. The `launch.manage_config_file` function can be used to add a config file as an input to the job and apply input values to the file at runtime.
+The Launch SDK provides a way to manage input values stored in config files in the job code. The `launch.manage_config_file` function can be used to add a config file as an input to the job and apply input values to the file at runtime.
```
Comment (tristan-spaulding2) [inline docs/guides/launch/job-inputs.md]: @KyleGoyette I started to wonder if there was something duplicative here, can you take another read through of the end.

File: docs/guides/launch/job-inputs.md [hunk 3]
```diff
@@ -4,23 +4,20 @@ displayed_sidebar: default
 
 # Manage job inputs
 
-Add variable inputs to your job.
+The core experience of Launch is easily experimenting with different job inputs like hyperparameters and datasets, and routing these jobs to appropriate hardware.  Once a job is created, users beyond the original author can adjust these inputs via the W&B GUI or CLI.    For information on how job inputs can be set when launching from the CLI or UI, see the [Enqueue jobs](./add-job-to-queue.md) guide.
 
-## Intro
+This section describes how to programatically control the inputs that can be tweaked for a job.
```
Comment (KyleGoyette) [inline docs/guides/launch/job-inputs.md]: "programatically" -> "programmatically" missing an "m"

File: docs/guides/launch/job-inputs.md [hunk 4]
```diff
@@ -4,23 +4,20 @@ displayed_sidebar: default
 
 # Manage job inputs
 
-Add variable inputs to your job.
+The core experience of Launch is easily experimenting with different job inputs like hyperparameters and datasets, and routing these jobs to appropriate hardware.  Once a job is created, users beyond the original author can adjust these inputs via the W&B GUI or CLI.    For information on how job inputs can be set when launching from the CLI or UI, see the [Enqueue jobs](./add-job-to-queue.md) guide.
```
Comment (KyleGoyette) [inline docs/guides/launch/job-inputs.md]: Extra space complaint by the linter here, easy to fix.

File: docs/guides/launch/job-inputs.md [hunk 5]
```diff
@@ -4,23 +4,20 @@ displayed_sidebar: default
...
 
 :::info
-`launch` sdk functions require `wandb-core`. See the [`wandb-core` README](https://github.com/wandb/wandb/blob/main/core/README.md) for more information.
+Launch SDK functions require `wandb-core`. See the [`wandb-core` README](https://github.com/wandb/wandb/blob/main/core/README.md) for more information.
 :::
 
 ## Reconfigure the `Run` object
 
-The `Run` object returned by `wandb.init` in a job can be reconfigured, by default. The `launch` sdk provides a way to customize what parts of the `Run.config` object can be reconfigured when launching the job.
+The `Run` object returned by `wandb.init` in a job can be reconfigured, by default. The `launch` SDK provides a way to customize what parts of the `Run.config` object can be reconfigured when launching the job.
```
Comment (KyleGoyette) [inline docs/guides/launch/job-inputs.md]: For consistency with other parts of your edit: 

"`launch` SDK" -> Launch SDK

File: docs/guides/launch/job-inputs.md (hunk 1 above)
Comment (KyleGoyette) [reply docs/guides/launch/job-inputs.md]: Here's an example, HF Tune, here's a config file: https://github.com/huggingface/tune/blob/main/configs/benchmark.yaml

Comment (KyleGoyette) [reply docs/guides/launch/job-inputs.md]: Side note, "The Launch SDK can manage also" doesn't sound quite right.


What if we rephrase this and the following line:
"The Launch SDK also provides a way to manage input values stored in config files in the job code-- a common pattern in many deep learning and Large Language Model use cases. The `launch.manage_config_file` function can be used to add a config file as an input to the Launch job, giving you access to edit values within the config file when launching the job.

File: docs/guides/launch/job-inputs.md (hunk 2 above)
Comment (KyleGoyette) [reply docs/guides/launch/job-inputs.md]: Yeah these two consecutive paragraphs aren't meshing well. However they make a lot more sense if you drop:
"The Launch SDK provides a way to manage input values stored in config files in the job code."

See my above comment for a rewrite

**Review by KyleGoyette — COMMENTED** [review]
Rereading, I think i see the dupe you mentioned. Left a few comments to help with phrasing and consistency. Also worth fixing the Vale errors about the double spacing after periods.

File: docs/guides/launch/job-inputs.md [hunk 6]
```diff
@@ -4,9 +4,9 @@ displayed_sidebar: default
 
 # Manage job inputs
 
-The core experience of Launch is easily experimenting with different job inputs like hyperparameters and datasets, and routing these jobs to appropriate hardware.  Once a job is created, users beyond the original author can adjust these inputs via the W&B GUI or CLI.    For information on how job inputs can be set when launching from the CLI or UI, see the [Enqueue jobs](./add-job-to-queue.md) guide.
+The core experience of Launch is easily experimenting with different job inputs like hyperparameters and datasets, and routing these jobs to appropriate hardware. Once a job is created, users beyond the original author can adjust these inputs via the W&B GUI or CLI. For information on how job inputs can be set when launching from the CLI or UI, see the [Enqueue jobs](./add-job-to-queue.md) guide.
 
-This section describes how to programatically control the inputs that can be tweaked for a job.
+This section describes how to programmatically control the inputs that can be tweaked for a job.
```
Comment (KyleGoyette) [inline docs/guides/launch/job-inputs.md:9]: Consider: This section describes how to programmatically control the inputs that are configurable parameters when launching a job.

File: docs/guides/launch/job-inputs.md [hunk 7]
```diff
@@ -4,9 +4,9 @@ displayed_sidebar: default
...
 
 By default, W&B jobs capture the entire Run.config as the inputs to a job, but the Launch SDK provides a function to control select keys in the run config or to specify JSON or YAML files as inputs.
```
Comment (KyleGoyette) [inline docs/guides/launch/job-inputs.md]: Missed this earlier, but mind wrapping "Run.config" in this line with `` ticks?

File: docs/guides/launch/job-inputs.md [hunk 8]
```diff
@@ -90,9 +90,9 @@ to load the run config input values anywhere in the job code.
 
 ## Reconfigure a file
 
-The Launch SDK can manage also input values stored in YAML or JSON config files in the job code--a common approach in many deep learning and LLM use cases.
+The Launch SDK also provides a way to manage input values stored in config files in the job code.   This is a common pattern in many deep learning and large language model use cases, like this [HuggingFace Tune](https://github.com/huggingface/tune/blob/main/configs/benchmark.yaml) example using Hydra or this [Axolotl config](https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/examples/llama-3/qlora-fsdp-70b.yaml)).
```
Comment (KyleGoyette) [inline docs/guides/launch/job-inputs.md]: Vale is complaining about double spacing after periods. There's a ton of these errors sprinkled throughout the docs, but worth calling out and taking a pass over the PR to make sure we're not introducing too many more.

**Review by KyleGoyette — APPROVED** [review]
Approved with one suggestion and a note on linting.

**Review by jcrawley — APPROVED** [review]

**Pull request merged**